packages/integration/src/bitbucket/core.test.ts

```diff
@@ -249,6 +251,42 @@ describe('bitbucket core', () => {
         config,
       );
       expect(defaultBranch).toEqual('main');
-    });
+      });
+
+      it('return default branch for Bitbucket Server for bitbucket version 5.11', async () => {
+        const defaultBranchResponse = {
+          displayId: 'main',
+        };
+        worker.use(
+          rest.get(
+            'https://api.bitbucket.mycompany.net/rest/api/1.0/projects/backstage/repos/mock/default-branch',
+            (_, res, ctx) =>
+              res(
+                ctx.status(404),
+                ctx.set('Content-Type', 'application/json'),
+                ctx.json(defaultBranchResponse),
```

Comment:
Nit: If it's a 404, these are useless no :P
Just an empty response makes more sense.